tests/pytest/run_files/submit_job.sh

```diff
@@ -7,4 +7,4 @@ RUNDIR=/rundir
 ulimit -s unlimited
 cp $HEAD/FV3/fv3.exe $RUNDIR/fv3.exe
 cd $RUNDIR
-mpirun -np 6 --allow-run-as-root --mca btl_vader_single_copy_mechanism none --oversubscribe $RUNDIR/fv3.exe
+mpirun -np 6 $RUNDIR/fv3.exe
```

Comment:
Not sure I understand the reason why we can remove these options now.

Reply:
None of those flags are standard options for MPICH.  Leaving them (even if they don't cause the call to MPI to fail) is just confusing.

Reply:
Ok!

Reply:
Though options are required for us to run the model with openmpi, and I agree is confusing to leave them if using mpich.

Reply:
A duplicate of this file which is never used is in the root of the repo, you might want to delete that copy so it doesn't cause confusion.

Reply:
Or update it, actually I just noticed it's used by `run_docker.sh` and I'm not sure if it's used by anything else. It's just not used by the tests.